File: Android/SayHi/app/src/main/java/de/oklemenz/sayhi/model/TagQuery.java
package de.oklemenz.sayhi.model;

import java.io.Serializable;

/**
 * Created by Oliver Klemenz on 22.02.17.
 */

public class TagQuery implements Serializable {

    public boolean favorite = false;
    public boolean search = false;
    public String searchText = "";
    public boolean own = false;
    public String categoryKey = "";
    public boolean categoryStaged = false;
    public String name = "";
    public String langCode = "";
}

File: iOS/SayHi/SayHi/TagQuery.swift
//
//  TagQuery.swift
//  SayHi
//
//  Created by Klemenz, Oliver on 19.12.16.
//  Copyright © 2016 Klemenz, Oliver. All rights reserved.
//

import Foundation

struct TagQuery {
    
    var favorite: Bool = false
    var search: Bool = false
    var searchText: String = ""
    var own: Bool = false
    var categoryKey: String = ""
    var categoryStaged: Bool = false
    var name: String = ""
    var langCode: String = ""
}
